src/fidesops/graph/data_type.py

```diff
@@ -105,6 +133,13 @@ def empty_value(self) -> ObjectId:
         """Empty objectId value"""
         return ObjectId("000000000000000000000000")
 
+    def truncate(self, length: int, val: ObjectId) -> ObjectId:
+        """Truncates value to given length"""
+        logger.warning(
+            "Length truncation not supported for ObjectId data_type. Using original masked value instead for update query."
+        )
```

Comment:
Since all of these truncate methods, with the exception of `string`, are identical, they should probably be implemented in the base class and just get overridden for string.

Reply:
good point!